cmd/knoxite/repository.go

```diff
@@ -106,6 +115,26 @@ func executeRepoInit() error {
 	return nil
 }
 
+func executeRepoChangePassword() error {
+	r, err := openRepository(globalOpts.Repo, globalOpts.Password)
+	if err != nil {
+		return err
+	}
+
+	password, err := readPasswordTwice("Enter new password:", "Confirm password:")
+	if err != nil {
+		return err
+	}
+
+	err = r.ChangePassword(password)
+	if err != nil {
+		return err
+	}
+
+	fmt.Printf("Changed password successfully\n")
```

Comment:
You could use `fmt.Println` here and get rid of the `\n`.

Reply:
Since all the other functions use fmt.Printf, I wanted to keep consistency.